### PR TITLE
Add a first version of an Intro to Bowtie post / announcement.

### PR DESCRIPTION
**Pull request by Julian**
This is a first draft, I have to read through for gaps in content and awkwardness, but getting it up ASAP for comments, since it's already taken me too long to get it "done".

So yes, comments welcome please from anyone either on what's here or what's not here!

### CODE REVIEW

**Comment by gregsdennis** [comment]
I feel like there needs to be some mention of the [JSON Path Comparison project](https://cburgmer.github.io/json-path-comparison/).  You had created an [issue](https://github.com/json-schema-org/JSON-Schema-Test-Suite/issues/314) for this idea initially, but (a few months) later I shared a link to the JSON Path site as a working example in another space.  I don't know how much inspiration you drew from that, but to me, it seems like a catalyst.

**Comment by gregsdennis** [comment]
Title and header seem to not be rendered right
image

File: pages/posts/bowtie-intro.md
Comment (gregsdennis) [inline pages/posts/bowtie-intro.md:2]: "Introducing: Bowtie!" perhaps?

File: pages/posts/bowtie-intro.md
Comment (gregsdennis) [inline pages/posts/bowtie-intro.md:15]: then remove "Bowtie:" from here

**Review by jdesrosiers — APPROVED** [review]

File: pages/posts/bowtie-intro.md
Comment (gregsdennis) [inline pages/posts/bowtie-intro.md:43]: I'm not hung up on it, but the official spec is being developed at https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base.  Alternatively this link can be used.  Although, the Goessner post is more well-known.  Just an idea.  I leave it to your preference.

Comment (Julian) [reply pages/posts/bowtie-intro.md:43]: I have no preference, I took the link from the comparison page (which links to that Goessner one) -- happy to update it, I don't know anything about JSONPath's development but certainly trust you :D

Comment (Julian) [reply pages/posts/bowtie-intro.md]: (Done)

**Review by gregsdennis — APPROVED** [review]

**Comment by Julian** [comment]
Ben's happy to read this after it's live :D, hitting merge (even though the date is technically yesterday, but yeah whatever, lies and the internet, what better pairing!).

**Pull request merged**
